source/_slider.slim

```diff
@@ -0,0 +1,61 @@
+div id="carousel-hero" class="carousel slide" data-ride="carousel"
+  / Indicators 
+  ol class="carousel-indicators hero-indicators"
```

Comment:
two classes were added because the first is from bootstrap and the second one is necessary for adding my styles